### PR TITLE
Add command to delete line (vscode)

### LINKED ISSUE #73: Add command to delete line (vscode)

### PR DESCRIPTION
**Pull request by CiroMeiraLins**
Resolves #73

### CODE REVIEW

File: CheatSheets/vscode-cheatsheet.md [hunk 1]
```diff
@@ -54,6 +54,7 @@
 | `Shift + Alt + DownArrow` | Copy line down |
 | `Ctrl + Alt + UpArrow` | Add cursor above |
 | `Ctrl + Alt + DownArrow` | Add cursor below |
+| `Ctrl + Alt + K` | Delete line |
```
Comment (crescentpartha) [inline CheatSheets/vscode-cheatsheet.md:57]:
```suggestion
| `Ctrl + Shift + K` | Delete line |
```

**Review by crescentpartha — CHANGES_REQUESTED** [review]
Request for changes

**Review by crescentpartha — CHANGES_REQUESTED** [review]
Please, resolve the required changes.

File: CheatSheets/vscode-cheatsheet.md (hunk 1 above)
Comment (crescentpartha) [inline CheatSheets/vscode-cheatsheet.md:57]:
```suggestion
| `Ctrl + Shift + K` | Delete line |
```
Please, change this line of code.

**Comment by shrishti147** [comment]
To delete a line in vs code :-
On Windows : Shift + Alt + Up/Down.
On Mac : Shift + Option + Up/Down.
On Ubuntu : Ctrl + Shift + Alt + Up/Down.

**Review by crescentpartha — APPROVED** [review]

**Pull request merged**